### PR TITLE
feat: lazy-load non-swap routes

### PR DESCRIPTION
**Pull request by zzmp**
## Description

Lazy load all non-swap pages to improve pageload.

This may make non-swap pages load slower on first load, but should be a minor change, and will improve pageload overall.

## Testing

To test:
- Navigate to the preview url
- Open your devtools (F12)
- In the Network pane, set throttling to "Fast 3G"
- Navigate to another page through the NavBar

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -14,44 +18,40 @@ import { useRouterPreference } from 'state/user/hooks'
...
+const NotFound = lazy(() => import('pages/NotFound'))
+const Pool = lazy(() => import('pages/Pool'))
+const PositionPage = lazy(() => import('pages/Pool/PositionPage'))
+const PoolV2 = lazy(() => import('pages/Pool/v2'))
+const PoolDetails = lazy(() => import('pages/PoolDetails'))
+const PoolFinder = lazy(() => import('pages/PoolFinder'))
+const RemoveLiquidity = lazy(() => import('pages/RemoveLiquidity'))
+const RemoveLiquidityV3 = lazy(() => import('pages/RemoveLiquidity/V3'))
+const TokenDetails = lazy(() => import('pages/TokenDetails'))
+const Tokens = lazy(() => import('pages/Tokens'))
```
Comment (just-toby) [inline src/pages/App.tsx:53]: are these low-traffic enough to make lazy?

**Review by just-toby — APPROVED** [review]

**Review by cbachmeier — APPROVED** [review]

**Comment by cbachmeier** [comment]
great branch name

File: src/pages/App.tsx (hunk 1 above)
Comment (zzmp) [reply src/pages/App.tsx:53]: i'm not making a very informed decision, just making the "normal" landing pages eager and the rest lazy.

**Review by pp-hh-ii-ll — APPROVED** [review]
LGTM

**Pull request merged**